var/spack/repos/builtin/packages/py-lib-pod5/package.py

```diff
@@ -0,0 +1,191 @@
+# Copyright 2013-2024 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+import platform
+import sys
+
+from spack.package import *
+
+arch, os = platform.machine(), sys.platform
+arch64_32, _ = platform.architecture()
+
+
+class PyLibPod5(PythonPackage):
+    """
+    POD5 is a file format for storing nanopore dna data in an easily accessible way.
+    The format is able to be written in a streaming manner which allows a sequencing
+    instrument to directly write the format.
+    """
+
+    homepage = "https://github.com/nanoporetech/pod5-file-format"
+    url = "https://files.pythonhosted.org/packages/bd/67/c1720a8e2ccd4442f49320dc238aa5ad5d92266f0aec21e50f5f42d392a6/lib_pod5-0.3.10-cp312-cp312-manylinux_2_17_x86_64.manylinux2014_x86_64.whl"
+
+    maintainers("Pandapip1")
+
+    license("MPL-2.0", checked_by="Pandapip1")
+
+    if (arch == "x86_64" or arch == "x64") and os == "linux":  # Linux on x86_64
```

Comment:
We should build from source, using `CMake`. Any advice @adamjstewart ?

Reply:
Yes, we should download from GitHub, set the `build_directory` to `python/lib_pod5`, and try the normal `PythonPackage` base class. If that doesn't work, we may need to build with `CMakePackage` or depend on a new C++ package in addition to the Python package. Don't have time to help myself, but see https://spack.readthedocs.io/en/latest/build_systems/pythonpackage.html for the docs on this.